HotelApi/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Microsoft.Owin.Hosting;
+
+namespace HotelApi
+{
+    class Program
+    {
+        const string Url = "http://localhost:9000";
```

Comment:
Should have:

Why you have defined const outside the method if it will be used just inside one method?

Reply:
applied